speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/frontend/IndicatorConfig.cpp

```diff
@@ -75,6 +75,15 @@ tSoundData* IndicatorConfig::LoadSound(void* p_handle, std::string p_path)
     snprintf(sndPath, PATH_BUF_SIZE, SOUNDS_DIR_FORMAT, GfDataDir(), source);
     data->Path = sndPath;
 
+    data->Looping = strcmp(GfParmGetStr(p_handle, p_path.c_str(), PRM_ATTR_LOOPING, VAL_NO), VAL_YES) == 0;
+    data->LoopInterval = GfParmGetNum(p_handle, p_path.c_str(), PRM_ATTR_LOOP_INTERVAL, nullptr, 0);
```

Comment:
Isn't loop interval only relevant if looping is enabled, don't know if it's neccesary to read it if it's disabled.

Reply:
It is, but it also doesn't really hurt to read it in it's only used when looping is enabled